README.md

```diff
@@ -19,41 +20,32 @@
 
 ## Overview
 
-`gotagger` is a library and CLI
-for tagging releases in git repositories
-as part of a CI process.
-
-`gotagger` looks through the git history of the current commit
-for the latest semantic version.
-This becomes the "base" version.
-Then `gotagger` examines all of the commit messages
-between the current commit and the latest tag,
-to determine if the most significant change was a
-feature,
-bug fix,
-or breaking change
-per the [Conventional Commits] format.
-`gotagger` then increments the base version accordingly
-and print the new version.
+`gotagger` is a library and CLI for tagging releases in git repositories as part
+of a CI process.
```

Comment:
You're going to need to re-format all of this. See https://github.com/sassoftware/gotagger/blob/main/CONTRIBUTING.md#write-documentation

Reply:
Done